data/json/effects.json

```diff
@@ -2034,16 +2034,35 @@
     "id": "boomered",
     "name": [ "Boomered" ],
     "desc": [ "Your vision is obscured by a disgusting splatter of bile." ],
-    "apply_message": "You're covered in bile!",
+    "apply_message": "Bile gets in your eyes!",
     "rating": "bad",
-    "base_mods": { "per_mod": [ -3 ], "vomit_chance": [ 500 ] },
+    "resist_traits": [ "SAPROVORE", "SAPROPHAGE", "EATDEAD" ],
```

Comment:
```suggestion
    "resist_traits": [ "SAPROVORE", "SAPROPHAGE", "EATDEAD" ],
```
Should these traits resist when it's in your eyes?  I can understand eater of the dead not worrying about being covered in the below body by disgusting but I wonder if their eyes are automatically protected?

Reply:
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/94415528/3da85993-5824-472a-8d2f-64b7b6279e8e)

The resist traits are only supposed to protect from the health malus and vomit chance, but I double checked and they were also protecting from the perception debuff. I've fixed that. Saprovores/phages are still blinded, given -5 per, marked with boomer-stink, and at risk of getting pinkeye.